simplq/src/components/pages/Home/MyQueues.jsx

```diff
@@ -1,41 +1,39 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useCallback } from 'react';
 import DeleteIcon from '@material-ui/icons/Delete';
 import IconButton from '@material-ui/core/IconButton';
 import { useHistory } from 'react-router';
-import { useAuth0 } from '@auth0/auth0-react';
+import { useFetchQueues, useDeleteQueue, selectQueues } from 'store/queues';
+import { useDispatch, useSelector } from 'react-redux';
 import styles from './home.module.scss';
-import { QueueRequestFactory } from '../../../api/requestFactory';
-import useRequest from '../../../api/useRequest';
 
 export default () => {
   const history = useHistory();
-  const { requestMaker } = useRequest();
-  const [myQueues, setMyQueues] = useState([]);
-  const { isAuthenticated } = useAuth0();
+  const dispatch = useDispatch();
+  const queues = useSelector(selectQueues);
+  const fetchQueues = useCallback(useFetchQueues(), []);
```

Comment:
Use `useCallback()` only if the action is going to be dispatched in `useEffect()`.